rules/aws_cloudtrail_rules/aws_cloudtrail_unsuccessful_mfa_attempt.py

```diff
@@ -0,0 +1,16 @@
+from panther_base_helpers import deep_get, aws_rule_context
+
+def rule(event):
+    mfa_used = deep_get(event,"additionalEventData","MFAUsed")
```

Comment:
this needs to provide a default value as well, looks like the empty string would be appropriate here

Reply:
This has been addressed.